hack/setup-kind-in-ci.sh

```diff
@@ -0,0 +1,183 @@
+
+# If a specific version of Istio hasn't been provided, try and guess the right one
+# based on the Kiali branch being tested (TARGET_BRANCH) and the compatibility matrices:
+# https://kiali.io/docs/installation/installation-guide/prerequisites/
+# https://istio.io/latest/docs/releases/supported-releases/
+if [ "${TARGET_BRANCH}" == "v1.48" ]; then
+  ISTIO_VERSION="1.13"
+elif [ "${TARGET_BRANCH}" == "v1.36" ]; then
+  ISTIO_VERSION="1.10"
+elif [ "${TARGET_BRANCH}" == "v1.24" ]; then
+  ISTIO_VERSION="1.7"
+fi
+
+KIND_NODE_IMAGE=""
+if [ "${ISTIO_VERSION}" == "1.7" ]; then
+  KIND_NODE_IMAGE="kindest/node:v1.18.20@sha256:e3dca5e16116d11363e31639640042a9b1bd2c90f85717a7fc66be34089a8169"
+elif [ "${ISTIO_VERSION}" == "1.10" ]; then
+  KIND_NODE_IMAGE="kindest/node:v1.21.10@sha256:84709f09756ba4f863769bdcabe5edafc2ada72d3c8c44d6515fc581b66b029c"
+elif [ "${ISTIO_VERSION}" == "1.13" ]; then
+  KIND_NODE_IMAGE="kindest/node:v1.23.4@sha256:0e34f0d0fd448aa2f2819cfd74e99fe5793a6e4938b328f657c8e3f81ee0dfb9"
+fi
+
+# print out our settings for debug purposes
+cat <<EOM
+=== SETTINGS ===
+DORP=$DORP
+ISTIO_VERSION=$ISTIO_VERSION
+KIND_NODE_IMAGE=$KIND_NODE_IMAGE
+TARGET_BRANCH=$TARGET_BRANCH
+=== SETTINGS ===
+EOM
+
+infomsg "Make sure everything exists"
+which kubectl > /dev/null || (infomsg "kubectl executable is missing"; exit 1)
+which kind > /dev/null || (infomsg "kind executable is missing"; exit 1)
+which "${DORP}" > /dev/null || (infomsg "[$DORP] is not in the PATH"; exit 1)
+
+NODE_IMAGE_LINE=""
+if [ -n "${KIND_NODE_IMAGE}" ]; then
+  NODE_IMAGE_LINE="image: ${KIND_NODE_IMAGE}"
+fi
+infomsg "Kind cluster to be created with name [ci]"
+cat <<EOF | kind create cluster --name ci --config -
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+  - role: worker
+    ${NODE_IMAGE_LINE}
+EOF
+
+# When a new cluster is created, kind automagically sets the kube context.
+
+infomsg "Create Kind LoadBalancer via MetalLB"
+lb_addr_range="255.70-255.84"
+
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/metallb.yaml
+
+subnet=$(${DORP} network inspect kind --format '{{(index .IPAM.Config 0).Subnet}}')
+subnet_trimmed=$(echo "${subnet}" | sed -E 's/([0-9]+\.[0-9]+)\.[0-9]+\..*/\1/')
+first_ip="${subnet_trimmed}.$(echo "${lb_addr_range}" | cut -d '-' -f 1)"
+last_ip="${subnet_trimmed}.$(echo "${lb_addr_range}" | cut -d '-' -f 2)"
+cat <<LBCONFIGMAP | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: default
+      protocol: layer2
+      addresses: ['${first_ip}-${last_ip}']
+LBCONFIGMAP
+
+if [ -n "${ISTIO_VERSION}" ]; then
+  DOWNLOAD_ISTIO_VERSION_ARG=" --istio-version ${ISTIO_VERSION}"
+fi
+
+infomsg "Downloading istio"
+hack/istio/download-istio.sh"${DOWNLOAD_ISTIO_VERSION_ARG}"
```

Comment:
found the problem - don't wrap this in quotes:
```suggestion
hack/istio/download-istio.sh ${DOWNLOAD_ISTIO_VERSION_ARG}
```

Reply:
bash ¯\_(ツ)_/¯

Reply:
Hmmm won't let me commit the suggestion since I force pushed so I'll just update it locally and push that